### DISCUSSION TITLE
Can you update an older draft?

### DISCUSSION THREAD

**Discussion by hmp3094** [release-drafter/release-drafter#1060] [Q&A]
Hello. Here is a scenario that I am hoping to use release drafter for.
There is a main develop branch to which changes are pushed. Every week, a new branch is created off of the develop branch with a semantic version number. That branch is then used to do all the release work. But there are times when let's say a bug fix is pushed to that newly created branch. 

Normally, the draft is updated with the changes that are pushed to the branches you have configured release drafter for and when you are ready to publish a new release, that draft is then closed. And the new draft is created the next time a change is pushed to the target branch(es). 

My question is whether there is a way to update the old draft with a change that has been pushed to the release branch after the fact. I know this is a complicated requirement but any help is appreciated. Thank you!

**Comment by jetersen** [comment]
see #148 and #1173 I plan to support this in the CLI version being created in #1204
